cmd/syft/main.go

```diff
@@ -15,4 +18,13 @@ func main() {
 	if err := cli.Execute(); err != nil {
 		log.Fatalf("error during command execution: %v", err)
 	}
+	f, err := os.Create("mem.pprof")
```

Comment:
note: this should be removed.

Also there should already be memory profiling built into syft. If you use `SYFT_DEV_PROFILE_MEM` --that being said, it seems to have been inadvertently removed. I'll try and restore this functionality.

Reply:
remove in commit f4df5a2e962d93e4ef4aa4defa393ac28b1938ba